tests/api/commands/test_flags.py

```diff
@@ -30,3 +27,66 @@ def _(password: str = PasswordOption):
     result = runner.invoke(app, ["--password", "dummy"], catch_exceptions=False)
     assert result.exit_code == 0
     assert PLAIN_PASSWORD_MSG in result.output
+
+
+def test_overrideable_option_is_overrideable(snapshot):
+    original = OverrideableOption(1, "--option", help="original help")
+    app = Typer()
+
+    @app.command()
+    def _(option: int = OverrideableOption(default=2, help="new help")):
+        return "ok"
+
+    runner = CliRunner()
+    result = runner.invoke(app, ["--help"], catch_exceptions=False)
+    assert result.exit_code == 0
+    assert result.output == snapshot
```

Comment:
Ah you're right, the test is intended to test that the new option has different properties than original, but I made a typo and reused OverrideableOption instead of original. Rewrote the test.